lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+    "description": "Links to MicroPython history in the about dialog"
+  },
+  "micropython-source-code": {
+    "defaultMessage": "MicroPython <linkV1>source code for the micro:bit V1</linkV1>(codi font per la micro:bit V1) i <linkV2>micro:bit V2</linkV2>",
+    "description": "Links to the MicroPython source code in the about dialog"
+  },
+  "more-action": {
+    "defaultMessage": "Més",
+    "description": "More button text (showing more/further content)"
+  },
+  "more-connect-options": {
+    "defaultMessage": "Més opcions de connexió",
+    "description": "Aria label for the additional actions menu to the right of the Send to micro:bit button"
+  },
+  "more-ideas": {
+    "defaultMessage": "<link>More ideas</link> Més idees.",
+    "description": "Link to more ideas on the microbit.org website at the end of the ideas section"
+  },
+  "more-save-options": {
+    "defaultMessage": "Més opcions de desar",
+    "description": "Aria label for the additional actions menu to the right of the Save button"
+  },
+  "multiple-files-message-one": {
+    "defaultMessage": "Aquest projecte conté els fitxers {fileCount, plural, other {{fileCount} files}}, però només s'ha baixat <strong>main.py</strong>.",
+    "description": "Message in dialog shown when multiple files are available for download. Uses ICU syntax for pluralisation: https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format. In practice, fileCount will always be two or more."
+  },
+  "multiple-files-message-two": {
+    "defaultMessage": "Fes servir el botó <strong>Desa</strong> per baixar tot el projecte com a fitxer hexadecimal. Pots <strong>obrir</strong> el fitxer hexadecimal per restaurar el teu projecte. Si prefereixes baixar fitxers Python individuals, fes servir la pestanya <strong>Projecte</strong>.",
+    "description": "Message in dialog shown when multiple files are available for download"
+  },
+  "multiple-files-title": {
+    "defaultMessage": "Avís: només s'ha descarregat main.py",
+    "description": "Title of dialog shown when multiple files are available for download but only the main file downloaded."
+  },
+  "name-project": {
+    "defaultMessage": "Anomena el teu projecte",
+    "description": "Name your project header"
+  },
+  "name-text": {
+    "defaultMessage": "Nom",
+    "description": "Header for name field"
+  },
+  "name-used-when": {
+    "defaultMessage": "El nom s'utilitza quan deses un fitxer hexadecimal.",
+    "description": "Text under project name field"
+  },
+  "new-file-hint": {
+    "defaultMessage": "T'afegirem l'extensió <code>.py</code>.",
+    "description": "Hint shown in the new Python file dialog"
+  },
+  "next-action": {
+    "defaultMessage": "Següent",
+    "description": "Next button text"
+  },
+  "not-found-checklist-one": {
+    "defaultMessage": "La teva  <strong>micro:bit està connectada</strong>? Has <link>follow these steps</link>? seguit aquests passos?",
```

Comment:
Again, translate the text inside <link> don't duplicate.

Reply:
Requested in Crowdin